fpx/src/commands/client/requests.rs

```diff
@@ -42,3 +48,78 @@ async fn get_request(args: GetArgs) -> Result<()> {
 
     Ok(())
 }
+
+/// Temporary inline this code into this module
+///
+#[derive(clap::Args, Debug)]
+pub struct ListArgs {
+    /// Base url of the fpx dev server.
+    #[arg(from_global)]
+    pub base_url: Url,
+}
+
+async fn list_requests(args: ListArgs) -> Result<()> {
+    let api_client = ApiClient::new(args.base_url.clone());
+
+    let request = api_client.request_list().await?;
+
+    serde_json::to_writer_pretty(stdout(), &request)?;
+
+    Ok(())
+}
+
+/// Temporary inline this code into this module
+///
+#[derive(clap::Args, Debug)]
+pub struct PostArgs {
+    /// Base url of the fpx dev server.
+    #[arg(from_global)]
+    pub base_url: Url,
+    /// HTTP method to use (e.g., GET, POST, PUT, DELETE).
```

Comment:
Can you add some newlines between de property and the documentation for some better readability